cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-sync.php

```diff
@@ -610,7 +610,7 @@ public function validate_sync_type( $type, $attachment_id ) {
 	 */
 	public function filter_status( $status, $attachment_id ) {
 
-		if ( $this->been_synced( $attachment_id ) || $this->is_pending( $attachment_id ) ) {
+		if ( $this->been_synced( $attachment_id ) && $this->is_pending( $attachment_id ) && $this->get_sync_type( $attachment_id ) ) {
```

Comment:
Calling `get_sync_type` outside the `if` may be a better idea to avoid calling the same function multiple times

Reply:
@dugajean - a good observation, but at this point, the `get_sync_type` will be using the cached version, meaning it will be run once on the page load. So it should represent the state at time of load.